src/main/java/test/project/bookshop/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package test.project.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
don't use Data on JPA entities, it contains ToSring, EqualsAndHashcode, which can be problematic, for now use @ Getter @ Setter.